src/Laserfiche.Repository.Api.Client.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <Version>1.1.2</Version>
+    <Version>1.1.3</Version>
```

Comment:
could you also update the version in the pipeline?

Reply:
ok done